keanu-project/src/main/java/io/improbable/keanu/util/io/ProtobufLoader.java

```diff
@@ -67,6 +75,124 @@ protected BayesianNetwork loadNetwork(SavedBayesNet.Graph graph) {
         return bayesNet;
     }
 
+    public Sequence loadSequence(InputStream inputStream) throws IOException {
+        Collection<Sequence> sequences = loadSequences(inputStream);
+        if (sequences.size() != 1) {
+            throw new SequenceConstructionException("The provided BayesianNetwork contains more than one Sequence");
```

Comment:
Absolutely, but in this case I think that the method loadSequence should never have been called in the first place. I see this as an exception worthy situation because the user will obviously be expecting a bayesnet with a sequence in it.